SkillChat.Client.ViewModel/MainWindowViewModel.cs

```diff
@@ -252,6 +267,7 @@ public MainWindowViewModel()
                         }
 
                         MessageReceived?.Invoke(new ReceivedMessageArgs(newMessage));
+                        messageDictionary.Add(newMessage.Id, newMessage);
```

Comment:
Вместо добавление через Add, лучше использовать индексатор, потому что при попытке добавить ключ второй раз будет исключение, а через индексатор просто перезапишется.
```suggestion
                        messageDictionary[newMessage.Id] = newMessage;
```